test/acceptance/transformers/plaintext_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule Acceptance.Transformers.PlaintextTests do
+  use ExUnit.Case, async: true
+
+  doctest Earmark.Transformers.Plaintext, import: true
+
+  describe "markdown_to_plaintext" do
+
+    test "remove basic markup" do
+      markdown = "*Emphasized* text."
+      plaintext = "Emphasized text."
+
+      assert as_plaintext(markdown) == {:ok, plaintext, []}
+    end
+
+    test "keep newlines while removing markup" do
+      markdown = "*Line one*\n[Line two](http://example.com)\nLine **three**"
+      plaintext = "Line one\nLine two\nLine three"
+
+      assert as_plaintext(markdown) == {:ok, plaintext, []}
+    end
+
+
+    test "an example with block elements" do
+      markdown = """
+      1. Item One
+      1. Item Two
+         > Block
+      """
+      plaintext = "Item OneItem TwoBlock"
```

Comment:
:thinking: ideally it would keep the newlines, at least not glue words together. I thought I had a test case for this.. strangely enough github gives me an error when I try to fully see your changes.

Reply:
sorry too many force pushes, can you make a PR based on this branch, but against master, to implement this issue?